mysql/datadog_checks/mysql/mysql.py

```diff
@@ -1,4 +1,4 @@
-# (C) Datadog, Inc. 2018
+# (C) Datadog, Inc. 2018-present
 # (C) Datadog, Inc. Patrick Galbraith <patg@patg.net> 2013
```

Comment:
Should this be `2013-present` too?

Reply:
Should be `earliest date of code commit`-present.

Reply:
```
# (C) Datadog, Inc. 2018-present
# (C) Datadog, Inc. Patrick Galbraith <patg@patg.net> 2013-present
```

?